src/test/ui/issue-47380.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0369]: binary operation `+` cannot be applied to type `&str`
+  --> $DIR/blah.rs:13:33
+   |
+13 |     println!("🦀🦀🦀🦀🦀"); let _a = b + ", World!";
+   |                                      ^^^^^^^^^^^^^^ `+` can't be used to concatenate two `&str` strings
+help: `to_owned()` can be used to create an owned `String` from a string reference. String concatenation appends the string on the right to the string on the left and may require reallocation. This requires ownership of the string on the left
+   |
+13 |     println!("🦀🦀🦀🦀🦀"); let _a = b.to_owned() + ", World!";
```

Comment:
Note for reviewers: this looks misaligned on the browser, but not in terminals that handle emojis correctly.

Reply:
@estebank the output is misaligned in iTerm though